apis/compute/validation/machine_test.go

```diff
@@ -55,10 +55,6 @@ var _ = Describe("Machine", func() {
 			&compute.Machine{},
 			ContainElement(RequiredField("spec.machineClassRef")),
 		),
-		Entry("no image",
```

Comment:
Why is this test removed? Isn't it actually verifying it's possible to have no image?